src/utils/sorting/Project.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  ChapterGetProjects,
+  NonprofitGetProjects,
+  SortingOption,
+} from "src/utils/types";
+
+export function chapterGetProjectsSorter(projectsGet: ChapterGetProjects) {
+  const sorter: { [key: string]: string } = {};
+
+  const createdAt = projectsGet.sortCreatedAt;
+  if (createdAt != undefined) {
+    if (createdAt == SortingOption.ASCENDING) {
+      sorter["createdAt"] = SortingOption.ASCENDING;
+    } else if (createdAt == SortingOption.DESCENDING) {
+      sorter["createdAt"] = SortingOption.DESCENDING;
+    }
+  }
+
+  const updatedAt = projectsGet.sortUpdatedAt;
+  if (updatedAt != undefined) {
+    if (updatedAt == SortingOption.ASCENDING) {
+      sorter["updatedAt"] = SortingOption.ASCENDING;
+    } else if (updatedAt == SortingOption.DESCENDING) {
+      sorter["updatedAt"] = SortingOption.DESCENDING;
+    }
+  }
+
+  const sortStatus = projectsGet.sortStatus;
```

Comment:
When sorting by project status, is it sorting by order or alphabetically? For example, if the two stages were "Schedule Interview" and "Maintenance", would "Maintenance" come first (alphabetical) or after (order). We want to sort based on the order, which might be the case here if mongoose handles enums differently but I just want to double check.